packages/ui-kit/src/components/rss/rss-feeds.spec.ts

```diff
@@ -1,4 +1,7 @@
-import { transformData } from './rss-feeds';
+/**
+ * @jest-environment jsdom
+ */
```

Comment:
What's this for?

Reply:
Yes but aren't we already using `jsdom`? See https://github.com/commercetools/commercetools-docs-kit/blob/890dc3db00f97434f0cdc60d3e0efcbcbbcb33e3/jest.test.config.js#L35

Reply:
turned out to be a follow-up error:  `jest-preset-gatsby` was not configured for typescript so the whole config was not picked up (I had tested only that file first and oversaw that the test is not running any more in the full test suite)